src/containers/other/NFTView/NFTView.tsx

```diff
@@ -142,6 +157,14 @@ const NFTView: React.FC<NFTViewProps> = ({ onTransfer }) => {
 				wildPriceUsd={wildPriceUsd}
 				refetch={refetch}
 				isBiddable={isBiddable}
+				bidData={allBids}
+				isLoading={isBidDataLoading}
+				domainMetadata={domainMetadata}
+				domain={domainData}
+				setIsViewBidsOpen={setIsViewBidsOpen}
```

Comment:
Maybe it would be better to pull the Overlays up to NFTView and handle it from there rather than passing more props down?

Reply:
It could even be worth making a hook/provider with Context API for controlling the modals - would reduce complexity at the cost of slightly tighter coupling. Just one called `NFTViewModalProvider` to use locally on the `NFTView` component would be quite handy, as there's a *lot* of prop passing going on here.

However, I don't think you'll need to pass down state for Buy Now and Set Buy Now, as the wrapper buttons should come with internal show/hide modal logic.